bazel/test.bzl

```diff
@@ -353,7 +353,10 @@ def redpanda_cc_bench(
         duration = None,
         data = [],
         tags = [],
-        target_compatible_with = []):
+        target_compatible_with = [],
+        # If True, redirects stdout (seastar logging, mostly) to a
+        # file so that it does not overwhelm the result output
```

Comment:
nit: don't really need this comment with the docstring.

Reply:
Yes, I meant to remove it. Fixed.